doc/src/manual/interfaces.md

```diff
@@ -199,10 +199,10 @@ ourselves, we can officially define it as a subtype of an [`AbstractArray`](@ref
 | `setindex!(A, v, i::Int)`                       |                                          | (if `IndexLinear`) Scalar indexed assignment                                           |
 | `setindex!(A, v, I::Vararg{Int, N})`            |                                          | (if `IndexCartesian`, where `N = ndims(A)`) N-dimensional scalar indexed assignment       |
 | **Optional methods**                            | **Default definition**                   | **Brief description**                                                                 |
-| `IndexStyle(::Type)`                            | `IndexCartesian()`                       | Returns either `IndexLinear()` or `IndexCartesian()`. See the description below.      |
```

Comment:
I think parentheses are important here, as they indicate an instance should be returned rather than a type.